packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -983,7 +939,7 @@ describe('<Tooltip />', () => {
 
   describe('prop: followCursor', () => {
     it('should use the position of the mouse', async function test() {
-      // Only callig render() outputs:
+      // Only calling render() outputs:
       // An update to ForwardRef(Popper) inside a test was not wrapped in act(...).
```

Comment:
This is triggered due to the async update in Tooltip.js. We can fix this using .forceUpdate() but I'm not sure about using synchronous update in mouseMove.